packages/plugins/i18n/admin/src/components/CheckboxConfirmation/index.js

```diff
@@ -64,21 +64,21 @@ const CheckboxConfirmation = ({ description, isCreating, intlLabel, name, onChan
           <DialogBody icon={<ExclamationMarkCircle />}>
             <Stack size={2}>
               <Flex justifyContent="center" style={{ textAlign: 'center' }}>
```

Comment:
I don't really understand why we need that textalign center. Can't we do that with flex properties ?